networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/gossip/AttestationTopicSubscriptions.java

```diff
@@ -38,6 +47,7 @@ public synchronized void subscribeToCommittee(
     final UnsignedLong currentUnsubscribeSlot =
         unsubscriptionSlotByCommittee.getOrDefault(committeeIndex, ZERO);
     unsubscriptionSlotByCommittee.put(committeeIndex, max(currentUnsubscribeSlot, aggregationSlot));
+    handleCommitteeSubscriptionENRChange(committeeIndex);
```

Comment:
My thinking was that, since we don't have the random selection of persistent subnets yet, for now, the new ENR attestation subnet bitfield functionality would be used to display attestation subnets we're subscribing for aggregation. That was probably an assumption I should have made more explicitly, and maybe not the best assumption to make. I will refactor this PR to explicitly expose an interface to subscribe and unsubscribe from randomly selected persistent attestation subnets.